packages/browser/src/utils/event-utils.ts

```diff
@@ -78,32 +78,34 @@ export const COOKIE_CAMPAIGN_PARAMS = [
     'li_fat_id', // linkedin
 ]
 
-export function getCampaignParams(
-    customTrackedParams?: string[],
-    maskPersonalDataProperties?: boolean,
-    customPersonalDataProperties?: string[] | undefined
-): Record<string, string> {
+export function getCampaignParams(config?: PostHogConfig): Record<string, string> {
```

Comment:
I don't think `config` should be undefined here? AFAICS all call sites always have a config